src/commit-and-pull-request.sh

```diff
@@ -58,10 +58,14 @@ function git_commit_with_metadata {
   OVERLAYS: ${OVERLAY_NAMES}
   MANIFEST_JSON: ${MANIFEST_JSON}"
 
-  git commit -m "${TITLE}
+  if [ -n "${GIT_COMMIT_MESSAGE}" ]; then
```

Comment:
I don't see `GIT_COMMIT_MESSAGE` defined in the code

Reply:
https://github.com/kustomize-everything/action-promote/blob/5ac6e8b20fd94e42cae45bdab80b9ef193c9971b/action.yml#L142